src/Libraries/DSOfficeNodes/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,12 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("DSOffice")]
+[assembly: AssemblyCulture("")]
+
+[assembly: InternalsVisibleTo("DynamoMSOfficeTests")]
+[assembly: TypeForwardedTo(typeof(DSOffice.WorkBook))]
+[assembly: TypeForwardedTo(typeof(DSOffice.WorkSheet))]
```

Comment:
I moved these types to the OfficeUtilities assembly because they are used there and they are not actually visible in Dynamo